framework/core/Utils.pm

```diff
@@ -285,18 +285,23 @@ C<log_ref> is provided, the captured output is stored in that variable.
 sub exec_cmd {
     @_ >= 2 or die $ARG_ERROR;
     my ($cmd, $descr, $log_ref) = @_;
+    my $start_time = time();
     print(STDERR substr($descr . '.'x75, 0, 75), " ");
     my $log = `$cmd`; my $ret = $?;
     $$log_ref = $log if defined $log_ref;
+    my $elapsed_time = $time() - $start_time;
```

Comment:
This should be time (not $time), right?

Reply:
Yes.  That was failing the CI job.  I have fixed it, and CI is now passing.